[BR][2.0.x] SCARA default example config broken (again)

https://circleci.com/gh/fiveangle/Marlin/82?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Just an FYI that SCARA default example config is broken:

```
use_example_configs SCARA
In file included from Marlin/src/HAL/HAL_AVR/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_AVR/HAL_spi_AVR.cpp:40:
Marlin/src/HAL/HAL_AVR/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
In file included from Marlin/src/HAL/HAL_AVR/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_AVR/MarlinSerial.cpp:36:
Marlin/src/HAL/HAL_AVR/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
In file included from Marlin/src/HAL/HAL_AVR/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_AVR/persistent_store_impl.cpp:5:
Marlin/src/HAL/HAL_AVR/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
In file included from Marlin/src/HAL/HAL_AVR/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_AVR/servo_AVR.cpp:56:
Marlin/src/HAL/HAL_AVR/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
In file included from Marlin/src/HAL/HAL_AVR/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_AVR/watchdog_AVR.cpp:25:
Marlin/src/HAL/HAL_AVR/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
In file included from Marlin/src/HAL/HAL_LPC1768/../../inc/MarlinConfig.h:34:0,
from Marlin/src/HAL/HAL_LPC1768/LPC1768_Servo.cpp:63:
Marlin/src/HAL/HAL_LPC1768/../../inc/SanityCheck.h:1166:4: error: #error "Enable USE_ZMIN_PLUG when homing Z to MIN."
#error "Enable USE_ZMIN_PLUG when homing Z to MIN."
^
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_AVR/HAL_spi_AVR.o] Error 1
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_AVR/MarlinSerial.o] Error 1
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_AVR/persistent_store_impl.o] Error 1
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_AVR/servo_AVR.o] Error 1
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_AVR/watchdog_AVR.o] Error 1
*** [.pioenvs/megaatmega2560/src/src/HAL/HAL_LPC1768/LPC1768_Servo.o] Error 1
========================== [ERROR] Took 3.52 seconds ==========================

================================== [SUMMARY] ==================================
Environment megaatmega2560          	[ERROR]
Environment megaatmega1280          	[SKIP]
Environment anet10                  	[SKIP]
Environment DUE                     	[SKIP]
Environment LPC1768                 	[SKIP]
Environment LPC1768_debug_and_upload	[SKIP]
Environment rambo                   	[SKIP]
Environment sanguino_atmega644p     	[SKIP]
Environment STM32F1                 	[SKIP]
Environment teensy20                	[SKIP]
Environment teensy35                	[SKIP]
========================== [ERROR] Took 3.53 seconds ==========================
Exited with code 1
```

I don't have SCARA so don't really care myself.  As I suspect SCARA is the least popular kinematic style, if no one else cares either, please close without resolution and I'll refrain from submitting these notices.  If a user encounters the problem they will hopefully submit an issue and not just give up.

Thanks,

-=dave

--- a/Marlin/example_configurations/SCARA/Configuration.h
+++ b/Marlin/example_configurations/SCARA/Configuration.h
@@ -484,10 +484,10 @@
 // extra connectors. Leave undefined any used for non-endstop and non-probe purposes.
 #define USE_XMIN_PLUG
 #define USE_YMIN_PLUG
-//#define USE_ZMIN_PLUG
+#define USE_ZMIN_PLUG
 //#define USE_XMAX_PLUG
 //#define USE_YMAX_PLUG
-#define USE_ZMAX_PLUG
+//#define USE_ZMAX_PLUG
 
 // coarse Endstop Settings
 //#define ENDSTOPPULLUPS // Comment this out (using // at the start of the line) to disable the endstop pullup resistors

